Packet++/src/RadiusLayer.cpp

```diff
@@ -243,6 +243,9 @@ bool RadiusLayer::isDataValid(const uint8_t* udpData, size_t udpDataLen)
 	if(udpData != NULL)

```

Comment:
Might be more simple

```suggestion
	if(udpData != NULL && udpDataLen > sizeof(radius_header))

```